### DISCUSSION TITLE
Empty screen With iReport and drag and drop problem With javafx scenebuilder

### DISCUSSION THREAD

**Discussion by filippoaceto22** [hyprwm/Hyprland#7561] [General]
Good evening, I need to use an old version of iReport with hyprland, but it gives me a blank screen. If I click at the top the dropdown menus open but the content of the window is empty.
I also have a problem with javafx scemebuilder, if I try to drag an item it does not drag but is released right away. Has this ever happened to anyone?

**Comment by Excal-rs** [comment]
For your javafx scenebuilder issue, adding the following line in your config should fix it.
```
windowrule = stayfocused,^(com.oracle.javafx.scenebuilder.app.SceneBuilderApp)$
```

Note that this means if a scenebuilder window is visible you can't use your cursor to change focus, you have to use the keyboard, but it fixes the issue.

**Reply by manuel3053** [reply]
Because of changes in windowrule, you have to write the following line in your config file

`
windowrule = stayfocused,class:com.oracle.javafx.scenebuilder.app.SceneBuilderApp
`
